### PR TITLE
Allow to use code coverage with PHPSpec 5.0.

### PR DESCRIPTION
**Pull request by shulard**
Hello,

I've created this little PR to upgrade the dependencies and allow to use this awesome CodeCoverage wrapper on latest library releases. 

This also allow to use the new PCov coverage driver which is lighter : https://github.com/krakjoe/pcov

### CODE REVIEW

File: src/Listener/CodeCoverageListener.php [hunk 1]
```diff
@@ -123,11 +121,9 @@ public function afterExample(ExampleEvent $event): void
      */
     public function afterSuite(SuiteEvent $event): void
     {
-        if (!$this->enabled || $this->skipCoverage) {
+        if ($this->skipCoverage) {
             if ($this->io && $this->io->isVerbose()) {
-                if (!$this->enabled) {
-                    $this->io->writeln('No code coverage will be generated as neither Xdebug nor phpdbg was detected.');
-                } elseif ($this->skipCoverage) {
+                if ($this->skipCoverage) {
```
Comment (andrzejkupczyk) [inline src/Listener/CodeCoverageListener.php]: This check is rather unnecessary since we already know that this is true.

Comment (andrzejkupczyk) [inline src/Listener/CodeCoverageListener.php]: This check is rather unnecessary since we already know that this is true.

**Review by andrzejkupczyk — CHANGES_REQUESTED** [review]

File: src/Listener/CodeCoverageListener.php (hunk 1 above)
Comment (shulard) [reply src/Listener/CodeCoverageListener.php]: Thank you for the review, I've updated the code and rebased on master.

**Review by andrzejkupczyk — APPROVED** [review]

**Review by bartosz-maciaszek — APPROVED** [review]

**Review by aur1mas — APPROVED** [review]

**Comment by Nyholm** [comment]
Friendly ping

**Comment by drupol** [comment]
Ping @ek9 :-)

**Review by drupol — APPROVED** [review]
I definitely give my :+1: for this.

**Comment by drupol** [comment]
@Nyholm I tried to ping @ek9 by email but I got an email back, it seems that the email is no more working, and the domain name as well.

The maintainer doesn't have any contribution in the last year (https://github.com/ek9).

Do you think we should fork this ?

**Comment by drupol** [comment]
Ok guys, I made the fork: https://packagist.org/packages/drupol/phpspec-code-coverage

As soon as we have news from the original maintainer I'll delete my fork.

**Pull request closed without merging**